lib/rules/require-capitalized-comments.js

```diff
@@ -66,25 +137,44 @@ module.exports.prototype = {
     check: function(file, errors) {
         var inTextBlock = null;
 
+        var exceptions = this._exceptions || {};
+        var hasExceptions = this._hasExceptions;
+
         var letterPattern = require('../../patterns/L');
         var upperCasePattern = require('../../patterns/Lu');
 
         file.getComments().forEach(function(comment) {
-            var stripped = comment.value.replace(/[\n\s\*]/g, '');
-            var firstChar = stripped[0];
-            var isLetter = firstChar && letterPattern.test(firstChar);
+            var firstChar;
+            var isException;
+            var isLetter;
+            var isUpperCase;
+            var isValid;
+            var stripped;
+
+            if (hasExceptions) {
+                // strip leading spaces and any asterisks
+                // split on whitespace and colons
+                var splitComment = comment.value.replace(/(^\s+|[\*])/g, '').split(/[\s\:]/g);
+
+                isException = !!exceptions[splitComment[0]];
+            }
+
+            stripped = comment.value.replace(/[\n\s\*]/g, '');
+            firstChar = stripped[0];
+            isLetter = firstChar && letterPattern.test(firstChar);
 
             if (!isLetter) {
                 inTextBlock = false;
                 return;
             }
 
-            var isUpperCase = upperCasePattern.test(firstChar);
-            var isValid = isUpperCase || (inTextBlock && !isUpperCase);
+            isUpperCase = upperCasePattern.test(firstChar);
+
+            isValid = isException || isUpperCase || (inTextBlock && !isUpperCase);
 
             if (!isValid) {
                 errors.add(
-                    'Comments must start with an uppercase letter, unless it is part of a textblock',
+                    'Comments must start with an uppercase letter, unless it is part of a textblock or is an exception',
```

Comment:
Shall I remove the text about the exception? Does it remain useful here, given the early return?